man/rgbasm-old.5

```diff
@@ -0,0 +1,333 @@
+'\" e
+.\"
+.\" SPDX-License-Identifier: MIT
+.\"
+.Dd October 21, 2024
+.Dt RGBASM-OLD 5
+.Os
+.Sh NAME
+.Nm rgbasm-old
+.Nd obsolete language documentation
+.Sh DESCRIPTION
+This is the list of features that have been removed from the
+.Xr rgbasm 5
+assembly language over its decades of evolution, along with their modern alternatives.
+Its goal is to be a reference for backwards incompatibility, when upgrading an old assembly codebase to work with the latest RGBDS release.
+It does
+.Em not
+attempt to list syntax bugs that were fixed, nor new reserved keywords that may conflict with old identifiers.
+.Sh REMOVED
+.Ss Global labels without colons
+Deprecated in 0.4.0, removed in 0.5.0.
+.Pp
+Labels used to be definable with just a name, but had to be typed in column 1.
+.Pp
+Instead, use explicit colons; for example,
+.Ql Label:
+or exported
+.Ql Label:: .
```

Comment:
Arguably this is `REPLACED`?

Reply:
I think of "Replaced" as "easily fixable with `sed`", and this isn't. (Just doing `/^([A-Za-z_][A-Za-z0-9_@#]*)(\s*;.*)?/` would also match column-1 instructions and directives.)

Reply:
Ah, I see. Feel free to add section descriptions to that effect.

Reply:
I did so. :)